asm_utils/src/reachability.rs

```diff
@@ -19,10 +20,11 @@ pub fn filter_reachable_from<R: Register, F: FunctionOpKind>(
         .iter()
         .map(|(k, v)| (k.as_str(), v.as_str()))
         .collect();
-    let referenced_labels = find_reachable_labels(label, statements, objects, &replacement_refs)
-        .into_iter()
-        .map(|s| s.to_owned())
-        .collect::<HashSet<_>>();
+    let referenced_labels =
+        find_reachable_labels::<R, F, A>(label, statements, objects, &replacement_refs)
```

Comment:
```suggestion
        find_reachable_labels::<_, _, A>(label, statements, objects, &replacement_refs)
```

Also 3 more occurrences in this file